dll/shellext/zipfldr/CZipExtract.cpp

```diff
@@ -591,10 +592,10 @@ class CZipExtract :
 
             bool is_dir = Name.GetLength() > 0 && Name[Name.GetLength()-1] == '/';
 
-            char CombinedPath[MAX_PATH * 2] = { 0 };
-            PathCombineA(CombinedPath, BaseDirectory, Name);
-            CStringA FullPath = CombinedPath;
-            FullPath.Replace('/', '\\');    /* SHPathPrepareForWriteA does not handle '/' */
+            WCHAR CombinedPath[MAX_PATH * 2] = { 0 };
+            PathCombineW(CombinedPath, BaseDirectory, Name);
+            CStringW FullPath = CombinedPath;
```

Comment:
Can't we directly use a single `CStringW FullPath` with a pre-sized buffer of `MAX_PATH * 2` characters, then give to `PathCombineW` the `FullPath.GetBuffer()` or something?
(maybe @learn-more knows what can be the best, CString-wise)

Reply:
Implemented.